### PR TITLE
Frmw1213 deprecaterxapp

### PR DESCRIPTION
**Pull request by cathysiller**
JIRA: https://jira.rax.io/browse/FRMW-1213

### LGTMs
- [x] Dev LGTM
- [x] Dev+Vidyo LGTM
- [x] Design LGTM

### CODE REVIEW

File: demo/examples/layout.grid.html [hunk 1]
```diff
@@ -1,505 +0,0 @@
-<div class="layout-demo">
```
Comment (CITguy) [inline demo/examples/layout.grid.html]: Why was this file removed?

File: demo/scripts/demoApp.directive.js [hunk 2]
```diff
@@ -72,10 +72,6 @@
             {
                 linkText: 'Utilities',
                 children: linksForModuleCategory('utilities')
-            },
-            { /* Deprecated in favor of Elements */
-                linkText: 'Components',
-                children: linksForModuleCategory('components')
```
Comment (CITguy) [inline demo/scripts/demoApp.directive.js]: ❤️

File: demo/examples/layout.grid.html (hunk 1 above)
Comment (cathysiller) [inline demo/examples/layout.grid.html]: I believe you and I figured out together that it is a duplicate -- that file is somewhere else in the code base (where it should be) but I would double check me !!

Comment (CITguy) [inline demo/examples/layout.grid.html]: ah yes, carry on

File: src/utilities/rxNotify/scripts/rxNotify.js [hunk 3]
```diff
@@ -83,7 +83,7 @@ angular.module('encore.ui.utilities')
  *
  * ## Using the Page Stack
  *
- * The default notification stack is added by default to the `rxPage` template (see {@link rxApp}), so it should be
+ * The default notification stack is added by default to the `rxPage` template (see rxApp), so it should be
```
Comment (parlarjb) [inline src/utilities/rxNotify/scripts/rxNotify.js]: Remove the rxPage/rxApp bit

**Comment by parlarjb** [comment]
Vidyo LGTM

**Comment by ElementE** [comment]
Design LGTM -_sorry for delay, new phone and had to find my recovery codes_

**Comment by CITguy** [comment]
away we go
